sound/soc/sof/topology.c

```diff
@@ -3130,6 +3130,8 @@ static int sof_link_unload(struct snd_soc_component *scomp,
 	case SOF_DAI_INTEL_SSP:
 	case SOF_DAI_INTEL_DMIC:
 	case SOF_DAI_INTEL_ALH:
+	case SOF_DAI_IMX_SAI:
+	case SOF_DAI_IMX_ESAI:
```

Comment:
maybe we should also fix the comment below, e.g.

/* no resource needs to be released for all cases above */

Reply:
> maybe we should also fix the comment below, e.g.
> 
> /* no resource needs to be released for all cases above */

You are right, will fix.

Reply:
@plbossart Fixed.